src/main/java/storage/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package storage;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String PATH_TO_OUTPUT_FILE = "src/main/resources/output.csv";
+    private static final String FIRST_DOC_ROW = "fruit,quantity";
+
+    @Override
+    public File writeToFile(List<String> fruitDataToWrite) {
```

Comment:
prepare report before writing, basically this method should only contain try-catch construction and receive fully prepared data as parameter